packages/main/src/MessageStrip.js

```diff
@@ -85,7 +85,7 @@ const metadata = {
 		 *
 		 * See all the available icons in the <ui5-link target="_blank" href="https://openui5.hana.ondemand.com/test-resources/sap/m/demokit/iconExplorer/webapp/index.html" class="api-table-content-cell-link">Icon Explorer</ui5-link>.
 		 *
-		 * @type {string}
+		 * @type {HTMLElement}
```

Comment:
can you please add @ slot and remove the @ defaultvalue

Reply:
done. is `@type HTMLElement` fine or should I use `Node` here as for the default slot?

Reply:
HTMLElement is fine